vendor/github.com/socketplane/libovsdb/client.go

```diff
@@ -82,7 +142,7 @@ func ConnectUsingProtocol(protocol string, target string) (*OvsdbClient, error)
 }
 
 // Connect creates an OVSDB connection and returns and OvsdbClient
-func Connect(ipAddr string, port int) (*OvsdbClient, error) {
+func Connect(ipAddr string, port int, protocol string) (*OvsdbClient, error) {
```

Comment:
Add comments for this function regarding supported protocols and which envs are being used for cert files, so that the caller of this library knows about it

Reply:
done